share/completions/gacutil.fish

```diff
@@ -0,0 +1,36 @@
+function __fish_seen_tokens -d 'Check whether there are already specified tokens in command line buffer'
```

Comment:
If we keep this function, it should be in `share/functions/__fish_seen_tokens` so it's close to `share/functions/__fish_seen_argument.fish` and `share/functions/__fish_contains_opt.fish`

Actually, I think we should replace it with something like `__fish_seen_argument -s i -o il -s u -o ul -o ul -s l`